umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -18,16 +24,27 @@ By default, you can
 - connect to HTTP using the HTTP node
 
 By installing additional packages, you can:
-- connect to OPC-UA (node-red-controb-opcua)
-- ...
+- connect to OPC-UA (node-red-contrib-opcua)
+- connect to kafka (node-red-contrib-kafkajs)
+- connect to Siemens-S7 (node-red-contrib-s7)
+- connect to serial (node-red-node-serialport)
+
+You can additionally contextualize the data, using function or other different nodes do manipulate the 
+received data.
 
 ## How can I use it?
 
-See also tutorial on accessing Node-RED
+Node-red comes preinstalled as a microservice with the umh. Follow our [get started](/docs/getstarted/) guide, 
```

Comment:
Do a step-by-step tutorial here.

1. Open Node-RED
2. Use default node such as HTTP or install the flow of your protocol (see above for an exemplery list
3. Process the data and align it with the UMH datamodel. Create e.g., process value messages (link to historian feature and process values). 
4. Send the result to MQTT / Kafka (link to tutorial)